setup.py

```diff
@@ -1,15 +0,0 @@
-#!/usr/bin/env python
-
```

Comment:
Why is it safe to delete this file? Wouldn't it make the package unusable without poetry?